### PR TITLE
Initial Frontend commit

### PR DESCRIPTION
**Pull request by maxwalts**
### Changes
Added components for general layout in src > components > layout
Added components for the landing page / interest form in src > components > homepage 
created a layout that views all page components in App.js

### testing
I have been testing by compiling using npm start in the directory

### CODE REVIEW

File: src/components/homepage/AboutTeam.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        <div class="card">
+            <div class="card-body">
+                <h3>About Us</h3>
+                <p>Our names, contact</p>
+            </div>
+        </div>
+    )
+}
+
+export default AboutTeam
```
Comment (seanstapleton) [inline src/components/homepage/AboutTeam.js:14]: NP: semicolon after this

File: src/components/homepage/Homepage.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import InterestForm from './InterestForm'
+import Footer from '../layout/Footer'
+import ExplainProduct from './ExplainProduct'
+import MissionStatement from './MissionStatement'
+import UXinfo from './UXinfo'
+import BuyerInfo from './BuyerInfo'
+import AboutTeam from './AboutTeam'
+
+
+
```
Comment (seanstapleton) [inline src/components/homepage/Homepage.js:11]: NP: only two empty lines here

File: src/components/homepage/AboutTeam.js (hunk 1 above)
Comment (seanstapleton) [reply src/components/homepage/AboutTeam.js:14]: (repeat for other files)

**Comment by maxwalts** [comment]
> 1. Getting an error when I try to run (`Module not found: Can't resolve './App' in '/Users/seanstapleton/code/uxmarketplace/src'`). Looks like there should be an App.js in the `src/` directory that wasn't pushed.

```
git rm --cached public -r
git rm --cached src -r
```
solved this issue for me. Just logging this in case it comes up later.

**Comment by maxwalts** [comment]
this branch can be deleted. front-end1 includes changes

**Pull request closed without merging**